test/Microsoft.ML.TestFramework/BaseTestBaseline.cs

```diff
@@ -497,6 +497,9 @@ protected bool CheckEqualityFromPathsCore(string relPath, string basePath, strin
                     count++;
                     var inRange = GetNumbersFromFile(ref line1, ref line2, digitsOfPrecision);
 
+                    line1 = Regex.Replace(line1, @"\s+", String.Empty);
```

Comment:
this is removing all whitespace?

Reply:
yes removing the whitespace in that line. In some netcoreapp tests an extra space is getting added. i was not able to find a possible cause for that

Reply:
I don't feel this is the right fix. We should figure out why extra spaces are getting added.

---
In reply to: [236356019](https://github.com/dotnet/machinelearning/pull/1700#discussion_r236356019) [](ancestors = 236356019)

Reply:
i can remove this and disable the failing tests until we found the reason ? We need this ci soon because today again the netcoreapp master 3.0 build got broken and some new tests started failing

Reply:
How many tests fail for this?

If it is in the single digits, I guess that approach (disable the tests and log an issue to investigate) seems the lesser of 2 bad choices. My preference would be to figure out why the extra spaces are getting added.